frame/clearing-house/src/lib.rs

```diff
@@ -38,6 +51,9 @@
 //!
 //! ## Related Modules
 //!
+//! - `pallet-virtual-amm`
```

Comment:
I'd suggest changing it to `pallet-vamm`, since that's the name for the pallet in branch `create-vamm`